components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -47,71 +61,11 @@ const getChartOptions = (intl, startDate, endDate, hostCreatedAt) => {
   };
 };
 
-const getCategories = (intl, startDate, endDate, hostCreatedAt) => {
-  const numberOfDays = dayjs(startDate || hostCreatedAt).diff(dayjs(endDate), 'day');
-  if (numberOfDays <= 7) {
-    const startDay = startDate.getDay();
-    return [...new Array(7)].map((_, idx) =>
-      intl.formatDate(new Date(0, 0, idx + startDay), { weekday: 'long' }).toUpperCase(),
-    );
-  } else if (numberOfDays <= 365) {
-    const currentMonth = new Date().getMonth();
-    return [...new Array(12)].map((_, idx) =>
-      intl.formatDate(new Date(0, idx + currentMonth + 1), { month: 'short' }).toUpperCase(),
-    );
-  } else {
-    return [...new Array(6)].map((_, idx) =>
-      intl.formatDate(new Date(new Date().getFullYear() - 5 + idx, 0), { year: 'numeric' }).toUpperCase(),
-    );
-  }
-};
-
-const getCategoryType = (startDate, endDate, hostCreatedAt) => {
-  const numberOfDays = dayjs(startDate || hostCreatedAt).diff(dayjs(endDate), 'day');
-  if (numberOfDays <= 7) {
-    return 'WEEK';
-  } else if (numberOfDays <= 365) {
-    return 'MONTH';
-  } else {
-    return 'YEAR';
-  }
-};
-
 const constructChartDataPoints = (category, dataPoints) => {
-  let chartDataPoints;
-
-  // Show data for the past 5 years from the current year.
-  if (category === 'YEAR') {
-    chartDataPoints = new Array(6).fill(0);
-    const currentYear = new Date().getFullYear();
-    dataPoints.forEach(dataPoint => {
-      const year = new Date(dataPoint.date).getFullYear();
-      if (year > currentYear - 6) {
-        chartDataPoints[5 - (currentYear - year)] = Math.abs(dataPoint.amount.value);
-      }
-    });
-    // Show data for the past 12 months
-  } else if (category === 'MONTH') {
-    chartDataPoints = new Array(12).fill(0);
-    dataPoints.forEach(dataPoint => {
-      const date = new Date(dataPoint.date);
-      const today = new Date();
-      if (today.getFullYear() - date.getFullYear() <= 1) {
-        chartDataPoints[(date.getMonth() + (12 - today.getMonth())) % 12] = Math.abs(dataPoint.amount.value);
-      }
-    });
-    // Show data for the past 7 days
-  } else if (category === 'WEEK') {
-    chartDataPoints = new Array(7).fill(0);
-    dataPoints.forEach(dataPoint => {
-      const date = new Date(dataPoint.date);
-      const today = new Date();
-      if (today.getFullYear() === date.getFullYear() && today.getMonth() === date.getMonth()) {
-        chartDataPoints[date.getDay() % 7] = Math.abs(dataPoint.amount.value);
-      }
-    });
-  }
-
+  const chartDataPoints = [];
+  dataPoints.forEach(dataPoint => {
+    chartDataPoints.push({ x: dataPoint.date, y: Math.abs(dataPoint.amount.value) });
+  });
   return chartDataPoints;
```

Comment:
:ice_cream: Looks like this can be simplified

```suggestion
  return dataPoints.map(({ date, amount }) => ({ x: date, y: Math.abs(amount.value) }));
```